stable/elasticsearch/templates/_helpers.tpl

```diff
@@ -96,6 +96,8 @@ plugin installer template
     - "sh"
     - "-c"
     - |
+      rm -rf /usr/share/elasticsearch/plugins/  # Fixes DEVOPS-2040
```

Comment:
What's the point in referencing internal identifier in public place.

Reply:
Good point, thanks. I'll fix it in later release.